test/Rings/MPolyAnyMap/MPolyRing.jl

```diff
@@ -124,7 +124,7 @@
     Kxz, (z1, z2) = Kx[:z1, :z2];
     f = hom(Kxz, Kxz, hom(Kx, Kxz, [z1, z2]), [z2, z1])
     g = hom(Kxz, Kxz, hom(Kx, Kx, [y, x]), [z1 + 1, z2 + 1])
-    fg = @inferred f * g
+    fg = f * g
```

Comment:
@thofma : I had reasons to not allow `CompositeMap`s at some points within the schemes. To achieve this, I had to overwrite composition for some constellations. But given the genererality of allowable coefficient maps, I did not see how to make this type stable anymore. Do you strongly object to this regression here?